doc/whats_new/v1.3.rst

```diff
@@ -531,6 +531,10 @@ Changelog
   is enabled by specifying how to select infrequent categories with
   `min_frequency` or `max_categories`. :pr:`25677` by `Thomas Fan`_.
 
+- |Enhancement| Subsampling through the `subsample` parameter can now be used in
```

Comment:
If we are okay with BC breaking, then we need another entry under the "Changed Models" section.